edenai_apis/apis/amazon/amazon_image_api.py

```
@@ -739,6 +739,53 @@ def image__face_compare(
             standardized_response=FaceCompareDataClass(items=face_match_list),
         )
 
+    async def image__aface_compare(
+        self, file1: str, file2: str, file1_url: str = "", file2_url: str = "", **kwargs
```

Ruff (0.14.4)

Unused method argument: `file1_url`

(ARG002)